src/Npgsql.NetTopologySuite/NetTopologySuiteHandler.cs

```diff
@@ -274,74 +274,14 @@ Task INpgsqlTypeHandler<GeometryCollection>.Write(GeometryCollection value, Npgs
 
         Task WriteCore(IGeometry value, NpgsqlWriteBuffer buf, NpgsqlLengthCache lengthCache, NpgsqlParameter parameter, bool async)
         {
-            _writer.Write(value, new WriteStream(buf));
+            _writer.Write(value, buf.GetStream());
 #if NET45
             return Task.Delay(0);
```

Comment:
I know this isn't a part of this PR, but I just noticed it. This `#if` can be eliminated simply by returning `PGUtil.CompletedTask` which works on all target frameworks.